mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -180,47 +191,82 @@ public static boolean dropTable(Configuration conf, Properties props) {
   /**
    * Returns true if HiveCatalog is used
    * @param conf a Hadoop conf
+   * @param props the controlling properties
```

Comment:
What does "controlling properties" mean? The table's properties from the metastore?